hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmBucketArgs.java

```diff
@@ -199,15 +218,24 @@ public Builder setQuotaInNamespace(long quota) {
       return this;
     }
 
+    public Builder setDefaultReplicationCOnfig(
```

Comment:
Ah, this is where the new method is defined I mentioned above. Typo in the method name COnfig -> Config.